### ISSUE TITLE
Ctrl + F10 would NOT release my mouse

### ISSUE THREAD

**Issue by iokzero** [joncampbell123/dosbox-x#2900]
Trying Windows 98se on Dosbox-x, but once I capture my mouse inside DOSbox-x, there's no way to get my mouse coursor out of it. except Ctrl + F9, which will close the whole program. 

I am running lastest DOSbox-x under Manjaro AUR on the lastest xFce desktop enivornment.

**Comment by joncampbell123** [comment]
Linux SDL1 seems to require CTRL+SHIFT for mapper shortcuts marked to respond to CTRL. Does CTRL+SHIFT+F10 work?

**Comment by iokzero** [comment]
Not really, tried every possible combination with SHIFT, CTRL, SUPER, ALT.
Nothing works so far. On the other hand CTRL + F9 works just fine.

Not sure if this would also happen on other Linus distro / Desktop
Environment though.

**Comment by maron2000** [comment]
CTRL+F10 working flawlessly here on Ubuntu 18.04 (both with and without Wayland).
0.83.18 SDL1 64bit Commit 88370683291f5c681d2a7d6fff088b947b783bd4
Haven't tried running Windows98 though.

**Comment by maron2000** [comment]
I confirmed as @joncampbell123 said that CTRL +SHIFT + F10 works but not CTRL + F10 with Manjaro XFCE 21.1.2 on VMWare VM.
This phenomena is not a bug of DOSBox-X, but a conflict with XFCE default shortcut keys.

I confirmed that by changing the default settings of global shortcut key of XFCE, CTRL+F10 works on both SDL1 & SDL2 builds with commit https://github.com/joncampbell123/dosbox-x/commit/88370683291f5c681d2a7d6fff088b947b783bd4

The following link for details.
https://forum.manjaro.org/t/keyboard-shortcuts-ctrl-fn-conflicts/38259
